UI/window-projector.cpp

```diff
@@ -451,17 +451,17 @@ void OBSProjector::OBSRenderMultiview(void *data, uint32_t cx, uint32_t cy)
 		qiX = sourceX + 4.0f;
 		qiY = sourceY + 4.0f;
 
-		/* ----------- */
```

Comment:
Again, just wanted to note that you removed the segmentation comment here.  It also doesn't appear to be related to this particular commit.